### DISCUSSION TITLE
Osu teleports my mouse cursor.

### DISCUSSION THREAD

**Discussion by Ekischleki** [ppy/osu#17051] [Q&A]
I noticed a bug that if you move your mouse pointer all the way up the screen that it randomly teleports.

https://user-images.githubusercontent.com/66246570/156367149-ddc197c0-ed36-440f-8a4a-98035083f206.mp4

**Comment by frenzibyte** [comment]
Please have a read through https://github.com/ppy/osu/blob/c197e27e33bc77da7b96990dc10d2677eda38cc4/CONTRIBUTING.md#i-would-like-to-submit-an-issue, specifically the second bullet regarding providing more details.

**Comment by peppy** [comment]
This is likely an SDL issue which was fixed but then regressed as the fix was undone. See https://github.com/libsdl-org/SDL/issues/5271
